### ISSUE TITLE
Question on nuPlan Cross-City Test Set Selection

### ISSUE THREAD

**Issue by Peterzhoujr** [vita-epfl/UniTraj#52]
Hi UniTraj team,thank you for your work! I’m exploring your nuPlan cross-city experiments and have two questions:

1.For cross-city evaluation, should i use the official nuPlan test/val split? I noticed the provided test set mixes all cities together—or should i instead carve out a separate per-city subset from the nuPlan train dataset to serve as my test set?

2.For training, the paper states that you use 10K samples per city—should I therefore set the max_data_num parameter in the config to 10,000 when running the training code?

Thank you very much for your time and insights—I appreciate your help and look forward to your reply!

**Comment by Alan-LanFeng** [comment]
1. Yes. This is how it works: Metadrive can run real-time simulation with ScenarioNet data. And ScenarioNet data is exactly what UniTraj uses for the training and evaluation of prediction models. To achieve what you described, you probably need to reformat the code in 'base_dataset'.py into a data process function, which takes the ScenarioNet data as input and output the model input. 

2. I haven't tried to visualize with the nuplan-devkit so I'm not sure how to do that.

3. We simply draw the predicted trajectory without any simulation for verification. It's true that simulation based visualization will be more intuitive but we currently don't have that script
